README.md

```diff
@@ -1,43 +1,73 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
----
+# Google Cloud SDK Docker
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
+
+This is Docker image for the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+Image includes default command line tools:
+*  gcloud:  Default set of gcloud commands
+*  gsutil:  Cloud Storage Command Line Tool
+*  bq: BigQuery Command Line Tool
+
+## Supported tags and respective Dockerfile links
+
+* 155.0.0, latest (Dockerfile)
 
 ## Usage
 
-Follow these instructions if you are running docker *outside* of Google Compute Engine:
+To use the prebuilt **Alpine** based image, pull and run from
+
+* [https://hub.docker.com/r/google/cloud-sdk/](https://hub.docker.com/r/google/cloud-sdk/)
+
+The docker images do not have an ENTRYPOINT so you can call gcloud directly as shown below:
+
+```bash
+docker run -ti  google/cloud-sdk:latest gcloud version --format='value("Google Cloud SDK")'
```

Comment:
do you think your users will be primarily interested in "starting to use gcloud commands" or "checking version"? whatever is the primary purpose, it probably should be the information they can quickly get.